server/pulp/server/webservices/views/tasks.py

```diff
@@ -30,6 +34,15 @@ def task_serializer(task):
     return task
 
 
+class TaskSearchView(search.SearchView):
+    """
+    This view provides GET and POST searching on TaskStatus objects.
+    """
+    generate_json_response = staticmethod(generate_json_response_with_pulp_encoder)
```

Comment:
On 03/30/2015 05:21 PM, Austin wrote:
> generate_json_response = staticmethod(generate_json_response_with_pulp_encoder)
> 
> @rbarlow <https://github.com/rbarlow>, you mentioned plans to change this?

Yeah, I want to try to get rid of both the staticmethod() (if possible)
and rename this attribute. I suppose I'll get to that after bug week is
over.

-- 
Randy Barlow